docs/listings/hellomp2.txt

```diff
@@ -1,60 +1,45 @@
 [ReFrame Setup]
-  version:           3.10.0-dev.3+c22440c1
+  version:           4.0.0-dev.1+36ffa085
   command:           './bin/reframe -c tutorials/basics/hellomp/hellomp2.py -r'
-  launched by:       user@host
-  working directory: '/path/to/reframe'
-  settings file:     '/path/to/reframe/tutorials/config/settings.py'
-  check search path: '/path/to/reframe/tutorials/basics/hellomp/hellomp2.py'
-  stage directory:   '/path/to/reframe/stage'
-  output directory:  '/path/to/reframe/output'
+  launched by:       user@tresa.local
+  working directory: '/home/user/Repositories/reframe'
+  settings file:     '/home/user/Repositories/reframe/tutorials/config/settings.py'
+  check search path: '/home/user/Repositories/reframe/tutorials/basics/hellomp/hellomp2.py'
+  stage directory:   '/home/user/Repositories/reframe/stage'
+  output directory:  '/home/user/Repositories/reframe/output'
+  log files:         '/var/folders/h7/k7cgrdl13r996m4dmsvjq7v80000gp/T/rfm-xspkjmaz.log'
 
 [==========] Running 1 check(s)
-[==========] Started on Sat Jan 22 13:21:56 2022
+[==========] Started on Sat Oct  1 20:06:29 2022 
 
 [----------] start processing checks
-[ RUN      ] HelloThreadedExtendedTest @catalina:default+gnu
-[ RUN      ] HelloThreadedExtendedTest @catalina:default+clang
-[     FAIL ] (1/2) HelloThreadedExtendedTest @catalina:default+clang
-==> test failed during 'sanity': test staged in '/path/to/reframe/stage/catalina/default/clang/HelloThreadedExtendedTest'
-[     FAIL ] (2/2) HelloThreadedExtendedTest @catalina:default+gnu
-==> test failed during 'sanity': test staged in '/path/to/reframe/stage/catalina/default/gnu/HelloThreadedExtendedTest'
+[ RUN      ] HelloThreadedExtendedTest /4733a67d @catalina:default+gnu
+[ RUN      ] HelloThreadedExtendedTest /4733a67d @catalina:default+clang
+[     FAIL ] (1/2) HelloThreadedExtendedTest /4733a67d @catalina:default+gnu
+==> test failed during 'sanity': test staged in '/home/user/Repositories/reframe/stage/catalina/default/gnu/HelloThreadedExtendedTest'
+[       OK ] (2/2) HelloThreadedExtendedTest /4733a67d @catalina:default+clang
```

Comment:
This test must be passing by accident (?), I think it should also be failing.

Reply:
Indeed, that's a random success! :-) I could rerun it to get two failures.